pom.xml

```diff
@@ -77,7 +77,7 @@
         <dependency>
             <groupId>com.ibm.icu</groupId>
             <artifactId>icu4j</artifactId>
-            <version>53.1</version>
+            <version>59.1</version>
```

Comment:
open-source-parent manages this, but is out of sync with the internal parent.  Think its worth bumping and releasing internally?